boskos-release/0.1/samples/service-account.yaml

```diff
@@ -0,0 +1 @@
+boskos-acquire/0.1/examples/service-account.yaml
```

Comment:
Can you please change it to `boskos-acquire/0.1/samples/pipelinerun.yaml`

Reply:
Thanks for catching these @PuneetPunamiya !